### DISCUSSION TITLE
Project title poll

### DISCUSSION THREAD

**Discussion by bentshrimp** [AgainIoT/Open-Set-Go#42] [Polls]
We are using "Open-Set-go" in GitHub, "open-set-go" in url, "OpenSetGo" on page header as the title.
우린 GitHub엔 "Open-Set-Go", url에는 "open-set-go", 페이지 헤더엔 "OpenSetGo"를 쓰고 있습니다.
We should choose one of them for our formal project title.
우린 공식적인 프로젝트 이름으로 이들 중 하나를 선택해야 합니다.

If you have any other idea on our project title, please leave a comment!
프로젝트 이름에 대한 생각이 있으시면, 댓글로 남겨주세요!

**Comment by ymw0407** [comment]
I think Open-Set-Go is Best!!
@bentshrimp Lets finish this vote :smile_cat:

**Comment by hummingbbird** [comment]
Personally, I think it's better to capitalize the first letter of each word. so I think Open-Set-Go is the best 👍👍

**Comment by ymw0407** [comment]
Once all the mainainers have voted, I think we can confirm the project name as `Open-Set-Go` and finish the vote!
일단 모든 maintainer들은 투표를 진행하였기 때문에 프로젝트 이름은 Open-Set-Go로 확정하고 해당 투표는 끝내도 될것 같습니다!
